client/state/data-layer/wpcom-http/utils.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns response data from an HTTP request success action if available
+ *
+ * @param {Object} action may contain HTTP response data
+ * @returns {any|null} response data if available
```

Comment:
Is `any` valid? Should it be `*` ? Also, for multiple types, is the syntax valid or does it need to be surrounded with parentheses?

http://usejsdoc.org/tags-param.html#multiple-types-and-repeatable-parameters